concourse/pipelines/autodelete-cloudfoundry.yml

```diff
@@ -29,6 +50,16 @@ jobs:
         trigger: true
       - get: bosh-secrets
       - get: paas-cf
+      - get: cf-secrets
+      - get: cf-manifest
+      - get: bosh-CA
+
+      - task: get-cf-cli-config
+        file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
+      - task: remove-healthcheck-app
+        file: paas-cf/concourse/tasks/remove-healthcheck-app.yml
+
```

Comment:
Would it be more explicit and faster to only delete the service? Is that possible?